src/common/makeWclApiUrl.ts

```diff
@@ -1,12 +1,13 @@
 import makeApiUrl from './makeApiUrl';
+import { QueryParams } from './QueryParams';
 
-const WCL_API_KEY = process.env.REACT_APP_WCL_API_KEY || undefined;
-if (!WCL_API_KEY && process.env.NODE_ENV === 'development') {
+const WCL_API_KEY = process.env.REACT_APP_WCL_API_KEY ?? undefined;
```

Comment:
|| May be important here since `REACT_APP_WCL_API_KEY` is likely a string.

Reply:
I've always thought it was better to use nullish coalescing operators than logical disjunction. Therefore my change. 👍

Reply:
Sure, but in this case the desired behavior is to turn an empty string into undefined as well.

Reply:
Alright! Thanks for teaching me some new stuff. 👍 Guess we will disregard those changes in makeWclApiUrl.js then. Is there any other issues with the other contributions I've done?